### PR TITLE
Update headless documentation

### PR DESCRIPTION
**Pull request by jono-alderson**

### CODE REVIEW

File: docs/customization/apis/rest-api.md [hunk 1]
```diff
@@ -5,53 +5,67 @@ sidebar_label: REST API
 description: Get all of the metadata for a post or URL in a single request, and as part of WordPress' WP-JSON response.
 custom_edit_url: https://github.com/Yoast/developer-docs/edit/master/docs/customization/apis/rest-api.md
 ---
+import Alert from '@site/src/components/Alert';
 
-In Yoast SEO 14.0 we introduced a REST API endpoint that'll give you all the metadata you need for a specific URL. This will make it very easy for headless WordPress sites to use Yoast SEO for all their SEO meta output.
+<Alert>
+The 'raw' `json` data format described below is only available in Yoast SEO versions 16.7 and upwards.
```
Comment (jdevalk) [inline docs/customization/apis/rest-api.md:11]: Please refer to the RC for now here.

Comment (jono-alderson) [reply docs/customization/apis/rest-api.md:11]: In what sense, sorry?

Comment (jdevalk) [reply docs/customization/apis/rest-api.md:11]: Literally linking to it 😀

Comment (jono-alderson) [reply docs/customization/apis/rest-api.md:11]: Done.

**Pull request merged**